x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/AuthorizationService.java

```diff
@@ -326,7 +325,7 @@ private void checkOperatorPrivileges(Authentication authentication, String actio
             threadContext
         );
         if (operatorException != null) {
-            throw denialException(authentication, action, originalRequest, "because it requires operator privileges", operatorException);
+            throw actionRequiresOperatorPrivileges(authentication, action, originalRequest, operatorException);
```

Comment:
Similarly, I don't think this extra `actionRequiresOperatorPrivileges` is necessary and would prefer to drop it for direct invocation of `actionDenied`.